recipe/meta.yaml

```diff
@@ -152,15 +152,26 @@ outputs:
         - ray.autoscaler
 
   - name: ray-dashboard
+    build:
+      script:
+        - cd python/ray/new_dashboard/client
+        - npm install
+        - npm ci
+        - npm run build
+        # not sure why this seems to get copied on windows but not linux...
+        - cp -R ./build $SP_DIR/ray/new_dashboard/client/build  # [not win]
```

Comment:
conda build usually only picks up stuff installed into the host prefix. why is this being copied to the source dir?

Reply:
I'm trying to copy to the site-packages (`$SP_DIR` not `$SRC_DIR`), because I don't know how to copy into `build/bdist.linux-x86_64/egg/ray/...`, where all the other files are being copied, before being packaged in an egg, which then gets unpackaged in site-packages... (AFAICT)

Reply:
ack yes wrong env var sorry!